.nvmrc

```diff
@@ -0,0 +1 @@
+v12
```

Comment:
Do we need this file committed? No harm I guess

Reply:
I think it's useful to have it committed, gives you a hint when nothing compiles that you're on the wrong version of NodeJS.